src/components/toolbar/toolbar.scss

```diff
@@ -107,6 +107,19 @@ md-toolbar {
     margin: inherit;
   }
 
+  .md-toolbar-title {
+    display: flex;
+    flex: 1;
+    width: 0;
+
+    span {
```

Comment:
Since the only formatting this applies is the overflow, I wanted something that shouldn't apply any additional formatting.

I guess I'm afraid that if we switch it to `:first-child`, people might try to use a `<div>` or something else strange and it will mess up.

I'm not opposed to switching it, and I honestly can't put a finger on why I feel this way, but using the `<span>` just **_feels_** like a better solution to me :confused:

Thoughts?